app/(modelsMap)/models/page.tsx

```diff
@@ -0,0 +1,10 @@
+import { DehydratedPlatforms } from "Features/ERDDAP/hooks/DehydrateComponent"
+import { ModelingPage } from "./modeling"
+
+export default async function ModelingIndexPage() {
+  return (
+    <DehydratedPlatforms>
```

Comment:
Do we need `<DehydratedPlatforms>` here? We may want to dehydrate STAC data to speed up catalog loading at some point, but I don't think we have any platform data in this route.

I think `modeling.tsx` could be renamed to `page.tsx` with the exports tweaked.

Reply:
Great--Took out `<DehydratedPlatforms>` and moved the main `<ModelingPage/>` components into `Page.tsx` to keep auxiliary components out of the main route.